backend/__tests__/test_event/test_join_event.js

```diff
@@ -0,0 +1,36 @@
+const { create } = require('../../routes/event/join_event')
+const location = require('../../routes/location/create_location')
+const account = require('../../routes/account/register_account')
+const { createLunchspace } = require('../../routes/lunchspace/create_lunchspace')
+const { createMockDatabase, dropMockDatabase } = require('../../lib/database/mock')
+
+const testSpaceName = 'testspace'
+const testSpaceSubdomain = 'test-space-subdomain'
+const testLocationName = 'McBurger'
+const testLocationCoordinates = { lat: 20, long: 10 }
+const testFirstName = 'Max'
+const testLastName = 'Mustermann'
+const testEmail = 'max.mustermann@gmail.com'
+const testPassword = 'password'
+const testTime = '10:30'
+const testDate = '2018-10-12'
+
+let testUserId
+let testLunchspaceId
+let testLocationId
+
+describe('create join event', () => {
```

Comment:
More tests needed. Only the success case is tested. Multiple error cases exits: input wrong, already joined, location does not exits.